docs/products/opensearch/concepts/access_control.rst

```diff
@@ -0,0 +1,81 @@
+Access control
+==============
+
+Aiven for OpenSearch supports index-level access control lists (ACL) to control permissions.
+
+You can grant the following permissions:
+
+* ``deny``: no access
+* ``admin``: full access to APIs and documents 
+* ``readwrite``: full access to documents
+* ``read``: allow only searching and retrieving documents
+* ``write``: allow updating, adding, and deleting documents
+
+Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indexes that the permission applies to. Patterns are glob-style, where ``*`` matches any number of characters and ``?`` matches any character. 
+
+When multiple rules match, they are applied in the order listed above. If no rules match, access is denied.
+
+As an example, we can use the following set of rules:
```

Comment:
Add a subheading to split the example from the information

Reply:
Done ;)